frontend/TestRun/ResultsGraphs.svelte

```diff
@@ -172,48 +173,81 @@
     });
 </script>
 <div class="filters-container">
-    <div class="input-group input-group-sm mx-2">
-        <input type="date" class="form-control date-input" bind:value={startDate} on:change={() => fetchTestResults(test_id)} />
-        <input type="date" class="form-control date-input" bind:value={endDate} on:change={() => fetchTestResults(test_id)} />
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(1)}>Last Month</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(3)}>Last 3 Months</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(6)}>Last 6 Months</button>
+    <label class="mt-1">Time range:</label>
+    <div class="input-group input-group-inline input-group-sm mx-2">
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(1, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(1); showCustomInputs = false;}}>
+            Last Month
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(3, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(3); showCustomInputs = false;}}>
+            Last 3 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(6, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(6); showCustomInputs = false;}}>
+            Last 6 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {showCustomInputs ? 'active' : ''}"
```

Comment:
This is a good candidate to utilize Svelte reactivity pattern:

```svelte
<script>

     let dateRange = 3;
     $: setDateRange(dateRange);
</script>

<button class:active={dateRange === 3} on:click={() => dateRange == 3}>Last 3 Months</button>
```

Reply:
thanks, improved